bridge/cypress/integration/remediation-sequences.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { SequencesPage } from '../support/pageobjects/SequencesPage';
+
+describe('Sequences', () => {
+  const sequencePage = new SequencesPage();
+
+  beforeEach(() => {
+    cy.intercept('/api/v1/metadata', { fixture: 'metadata.mock' });
```

Comment:
Could you put this block with all interceptions into SequencesPage into a `interceptSequences` method? This also might be useful for other test suites.